operator/config/default/.gitignore

```diff
@@ -0,0 +1,4 @@
+
+# Ignore autogen patch file for webhooks
```

Comment:
i think we should also gitignore the cainjection files under `config/crd/patches`. those should also not be needed anymore.
so something like `cainjection_in_*.yaml` in another gitignore file over there